packages/snaps-execution-environments/CHANGELOG.md

```diff
@@ -7,14 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [0.31.0]
-### Uncategorized
-- Bump `@metamask/utils` to `5.0.0` and handle breaking changes ([#1276](https://github.com/MetaMask/snaps-monorepo/pull/1276))
-- Update workflows to match `MetaMask/core` repository ([#1254](https://github.com/MetaMask/snaps-monorepo/pull/1254))
-- Disallow requesting dynamic RPC methods ([#1256](https://github.com/MetaMask/snaps-monorepo/pull/1256))
-- Improvements to Webpack plugin ([#1225](https://github.com/MetaMask/snaps-monorepo/pull/1225))
-- Remove JSDOM from all packages ([#1241](https://github.com/MetaMask/snaps-monorepo/pull/1241))
-- Implement tests using `webdriverio` ([#1231](https://github.com/MetaMask/snaps-monorepo/pull/1231))
-- Target `ES2020` ([#1247](https://github.com/MetaMask/snaps-monorepo/pull/1247))
+### Changed
+- **BREAKING:** Target `ES2020` ([#1247](https://github.com/MetaMask/snaps-monorepo/pull/1247))
+- **BREAKING:** Disallow requesting dynamic RPC methods ([#1256](https://github.com/MetaMask/snaps-monorepo/pull/1256))
```

Comment:
Can we list the methods we're disallowing? So that it's a bit more explicit.

Reply:
done